content/pages/mua.md

```diff
@@ -84,7 +84,13 @@ it can be corrected.
 
 ## Auditing and Logging ##
 
-The WiCS Systems Committee keeps records ("logs") on our machines to
+The WiCS Systems Committee keeps records ("logs") on our machines to:
+
+1. Have diagnostic information to refer to when hardware and software issues arise
+2. Document machine activity
+3. Refer to for auditing purposes
+
+Logs can be found in the /var/logs directory on all machines
```

Comment:
This is true but not relevant, and also not all-inclusive. I'd remove that. I just told you that for flavour when you asked about logging :)